recipes/gtkwave/build.sh

```diff
@@ -0,0 +1,4 @@
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+./configure --prefix=$PREFIX --with-tcl=$PREFIX/lib --with-tk=$PREFIX/lib --disable-xz
```

Comment:
Why disable xz?

Reply:
It could not find it, I'm re-enabling it.